profile/profile.go

```diff
@@ -402,6 +402,8 @@ func (p *Profile) CheckValid() error {
 				if f.ID == 0 || functions[f.ID] != f {
 					return fmt.Errorf("inconsistent function %p: %d", f, f.ID)
 				}
+			} else {
+				return fmt.Errorf("profile has nil function in location line")
```

Comment:
Can we include any context in the error message that will help triage the error?

Reply:
Done